api/src/main.ts

```diff
@@ -46,8 +46,11 @@ async function bootstrap() {
       )
       .setVersion(version)
       .setBasePath('/')
-      .addOAuth2('password', '/api/v1/auth/token', '/api/v1/auth/token')
```

Comment:
@rahul-rathore-576 this seems to be some "change" in how it was before. Can't we preserve both parameters? Or they got same default values now?

Reply:
@evereq I have updated `addOAuth2` method as per latest version of `@nestjs/swagger`. Can you please review it?